VoltAir/Engine/android/AndroidActivity.cpp

```diff
@@ -609,4 +609,26 @@ void AndroidActivity::onStudentUpdateReceiver(JNIEnv* env, jobject obj, jstring
     engine->setValidLettersAndNumbers(letters);
 }
 
+QString AndroidActivity::validLetters()
```

Comment:
@diegowald Could you add some documentation to this method to explain what it does?